metabrainz/static/css/theme/boostrap/boostrap.less

```diff
@@ -35,7 +35,7 @@
 @import "thumbnails.less";
 @import "alerts.less";
 //@import "progress-bars.less";
-//@import "media.less";
+@import "media.less";
```

Comment:
\o That was my decision.
This is exactly the example use case for Bootstrap [media objects](https://getbootstrap.com/components/#media) and the previously used grid model styling ran into some issues.

When this site was initially designed someone commented that line out thus disabling the relevant LESS because it simply didn't use media objects at that time.